workers/unity/Assets/Fps/Scripts/GameLogic/FpsDriver.cs

```diff
@@ -61,6 +66,8 @@ private void OnEnable()
             Cursor.lockState = CursorLockMode.Locked;
             Cursor.visible = false;
             serverMovement.OnForcedRotation += OnForcedRotation;
+            health.OnRespawn += OnRespawn;
+            spatial = GetComponent<SpatialOSComponent>();
```

Comment:
Move this up to group with the other `GetComponent`

Reply:
done